client/my-sites/email/email-providers-stacked-comparison/index.tsx

```diff
@@ -0,0 +1,141 @@
+import {
+	GOOGLE_WORKSPACE_BUSINESS_STARTER_YEARLY,
+	TITAN_MAIL_MONTHLY_SLUG,
+} from '@automattic/calypso-products';
+import { withShoppingCart } from '@automattic/shopping-cart';
+import { TranslateResult, useTranslate } from 'i18n-calypso';
+import React, { FunctionComponent, ReactElement } from 'react';
+import { connect } from 'react-redux';
+import QueryProductsList from 'calypso/components/data/query-products-list';
+import QuerySiteDomains from 'calypso/components/data/query-site-domains';
+import { getSelectedDomain } from 'calypso/lib/domains';
+import { hasGSuiteSupportedDomain } from 'calypso/lib/gsuite';
+import withCartKey from 'calypso/my-sites/checkout/with-cart-key';
+import { getCurrentUserCurrencyCode } from 'calypso/state/currency-code/selectors';
+import { errorNotice } from 'calypso/state/notices/actions';
+import { NoticeOptions } from 'calypso/state/notices/types';
+import { getProductBySlug, getProductsList } from 'calypso/state/products-list/selectors';
+import canUserPurchaseGSuite from 'calypso/state/selectors/can-user-purchase-gsuite';
+import getCurrentRoute from 'calypso/state/selectors/get-current-route';
+import { getDomainsWithForwards } from 'calypso/state/selectors/get-email-forwards';
+import { fetchSiteDomains } from 'calypso/state/sites/domains/actions';
+import { getDomainsBySiteId, isRequestingSiteDomains } from 'calypso/state/sites/domains/selectors';
+import { getSelectedSite } from 'calypso/state/ui/selectors';
+import EmailProvidersStackedCard from './email-provider-stacked-card';
+import { professionalEmailCard } from './provider-cards/professional-email-card';
+
+import './style.scss';
+
+type Site = {
+	ID: number;
+	slug: string;
+};
+
+type EmailProvidersStackedComparisonProps = {
+	comparisonContext: string;
+	selectedDomainName: string;
+	source: string;
+	cartDomainName?: string;
+	selectedSite?: Site;
+};
+
+export interface ProviderCard {
+	additionalPriceInformation?: TranslateResult;
+	badge?: ReactElement;
+	buttonLabel?: TranslateResult;
+	children?: ReactElement;
+	description: TranslateResult;
+	detailsExpanded: boolean;
+	discount?: string;
+	expandButtonLabel: TranslateResult;
+	features: TranslateResult[];
+	footerBadge?: ReactElement;
+	formattedPrice: string;
+	formFields: ReactElement;
+	logo: ReactElement;
+	onExpandedChange: ( providerKey: string, expanded: boolean ) => void;
+	onButtonClick?: ( event: React.MouseEvent ) => void;
+	productName: TranslateResult;
+	providerKey: string;
+	showExpandButton: boolean;
+}
+
+const EmailProvidersStackedComparison: FunctionComponent< EmailProvidersStackedComparisonProps > = (
+	props
+) => {
+	const translate = useTranslate();
+	const professionalEmail: ProviderCard = professionalEmailCard;
```

Comment:
Maybe call this const something more specific? e.g. `professionalEmailCardProps` or `professionalEmailCardData`.